pkgs/tools/misc/rtz/default.nix

```diff
@@ -27,6 +27,12 @@ rustPlatform.buildRustPackage rec {
     };
   };
 
+  patches = [
+    # Use a vendored version of uitoipa-swagger-ui which does not require
+    # network access at build time.
+    ./vendored-uitoipa-swagger-ui.patch
```

Comment:
Seems like we could avoid the patch by fetching it ourselves in a FOD and setting SWAGGER_UI_DOWNLOAD_URL to a file: URL.

Reply:
Very interesting idea, let me try that.

Reply:
Done